airbyte-integrations/connectors/source-harvest/source_harvest/streams.py

```diff
@@ -96,7 +96,12 @@ def request_params(
         next_page_token: Mapping[str, Any] = None,
     ) -> MutableMapping[str, Any]:
         params = super().request_params(stream_state=stream_state, stream_slice=stream_slice, next_page_token=next_page_token)
-        replication_start_date = stream_state.get(self.cursor_field) or self._replication_start_date
+
+        replication_start_date = None
```

Comment:
Note that this code is not needed given that the CDK pins pendulum version to <3.0.0 but this will allow us to unpin the CDK version. I was able to test that by:
* Testing with previous version
    * `pip install pendulum==3.0.0` with previous code
        * integration tests fail
* Testing with previous version
    * Checking out this code
        * integration tests pass